src/test/java/nl/tudelft/ewi/devhub/webtests/IssuesTest.java

```diff
@@ -186,6 +198,15 @@ public void testAddRemoveLabel(){
 		assertEquals(0x00ccff, label.getColor());
 	}
 	
+	@After
+	public void deleteIssues(){
+		issues.findAllIssues(group.getRepository()).forEach(x -> {
+			Stream<? extends IssueComment> comments = issueComments.getMostRecentIssueComments(Lists.<GroupRepository>asList(group.getRepository(), new GroupRepository[0]),10);
+			comments.forEach(comment -> issueComments.delete(comment));
+			issues.delete(x);
```

Comment:
True, but I think being able to delete an issue progremattically without having to delete its contents first is not so bad at all. But in practise issues won't be removed anyway ;)